src/components/ha-sidebar.html

```diff
@@ -140,6 +141,19 @@
 
       <div class='divider'></div>
 
+      <paper-icon-item class='horizontal layout justified'>
```

Comment:
This should be in the configuration panel.

Reply:
I like the idea of having it further out of the way, but what if the user doesn't have the configuration panel enabled? This is still only a frontend configuration, and doesn't affect the server.

Reply:
Then they should enable the configuration panel. We can't be glueing things to the sidebar, it's already getting too busy. I want to move the push notifications (which is browser only too) to the configuration panel.

Reply:
OK, works for me.